Makefile

```diff
@@ -11,6 +11,12 @@ build:
 test: build/universal-robots
 	./build/universal-robots-test
 
+run-clang-tidy:
+	clang-tidy-19 --config-file ./.clang-tidy -p build ./src/viam/ur/module/*.[h,c]pp
```

Comment:
Per discussion, just pass the .cpp files to clang-tidy (and I think clang-check as well), since they are the only thing known to the compilation database. Also, add a `--header-filter` that selects only the headers under `viam/ur/module` since those are the only ones we are interested in.